tests/features/machine_translation.feature

```diff
@@ -32,8 +32,11 @@ Feature: Machine translation
       | state      | validated     |
       | collection | Brexit        |
 
+    When I am logged in as a moderator
     # Inside a collection sidebar, only custom pages, glossary terms (but not
     # the glossary overview) and the about page should be translatable.
+    And I go to the edit form of the "Brexit" collection
+    Then I should not see any Webtools eTrans elements
```

Comment:
I see we're banning also the entity delete forms, then why aren't we testing that too?